pubspec.yaml

```diff
@@ -3,14 +3,14 @@
 # found in the LICENSE file.
 
 name: process_runner
-version: 4.1.3
+version: 4.1.4
 description: A process invocation astraction for Dart that manages a multiprocess queue.
 homepage: https://github.com/google/process_runner
 
 dependencies:
   args: ^2.3.0
   async: ^2.5.0
-  file: ^6.1.0
+  file: ^7.0.0
```

Comment:
I would also leave these as `>=6.1.0 <8.0.0` where possible to ease the transition, given no change is needed here it must work with both :).